src/commands/login-use.ts

```diff
@@ -0,0 +1,27 @@
+import { Command } from "../command";
+import * as utils from "../utils";
+import * as auth from "../auth";
+import { FirebaseError } from "../error";
+
+module.exports = new Command("login:use <email>")
+  .description("Set the default account to use for this project directory")
+  .action((email: string, options: any) => {
+    const allAccounts = auth.getAllAccounts();
+    const accountExists = allAccounts.some((a) => a.user.email === email);
+    if (!accountExists) {
+      throw new FirebaseError(
+        `Account ${email} does not exist, run login:list to see valid accounts`,
+        { exit: 1 }
+      );
+    }
+
+    const projectDir = options.projectRoot as string | null;
```

Comment:
Why not just `options.projectRoot as string || ""`? (I'm not a fan of using `null` if I can help it, as you may have guessed by now)

Reply:
This is the actual type based on `detectProjectRoot`, the `as` is just because I don't want to use `any`.